### DISCUSSION TITLE
Docker - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#103] [Announcements]
# Docker - CS自学指南

CS自学指南

https://csdiy.wiki/%E5%BF%85%E5%AD%A6%E5%B7%A5%E5%85%B7/Docker/

**Comment by eshoyuan** [comment]
请问 "其余的云相关的课程如 Kubernetes 需要付费" 是指哪个, 可以给个网址嘛

**Reply by CS-Trekker** [reply]
https://kubernetes.io/

**Comment by guowei-gong** [comment]
推荐《Docker 深入浅出》https://book.douban.com/subject/30486354/

**Comment by buhuizuocaigou** [comment]
请问，docker 按照那个网站学下去就可以么还是说需要找到别的东西呢

**Comment by Reed-yang** [comment]
推荐看看这个就好 https://www.ruanyifeng.com/blog/2018/02/docker-tutorial.html

**Comment by LongMenJuZhen** [comment]
然而docker hub，令人感叹

**Comment by ada-ys325** [comment]
感谢指路

**Comment by Future32** [comment]
Coursera上也可以直接上KodeKloud的课

**Comment by lil-Journey** [comment]
自建日本节点服务器 月租5元不限流量 本人也在用 失效时可联系人工修复 qq:1165116287

**Comment by spyatsea** [comment]
这个课程的全称是KodeKloud《Docker Training Course for the Absolute Beginner》。

**Comment by aka556** [comment]
https://labex.io/zh/learn/docker 这个提供在线学习环境，但是后续https://yeasy.gitbook.io/docker_practice 这个也可以

**Reply by aka556** [reply]
后续会收费
